### ISSUE TITLE
Plugininstallation with composer

### ISSUE THREAD

**Issue by Nico-Schaefer-2111** [shopwareDowntown/downtown#345]
I just wanted to install a plugin, a colleague of mine wrote for shopware 6, with composer. While executing composer require, specially the post update command, it showed me an error: 

```
> [ ! -f install.lock ] || bin/console system:update:finish
Run Post Update

Get collection from directories

In MigrationCollectionLoader.php line 48:
                                        
  No source registered for "Shopware\"  
```
                                        
 Does someone has experience with this ?

**Comment by LiaraAlis** [comment]
We have the same problem.
After an update to Shopware 6.3.4.0 the post update no longer aborts with this message.

The update is merged in this repository with PR #346

**Comment by shyim** [comment]
Should be solved
